### DISCUSSION TITLE
What about playlists?

### DISCUSSION THREAD

**Discussion by mtsbh** [Marekkon5/onetagger#408] [Ideas]
Feature to add selected or filtered tracks to playlists would be great. Have you thought about it? Exporting later those playlists to *.m3u; *.m3u8 or *.pls. for easy import elsewhere.

**Comment by Marekkon5** [comment]
Added option to export .m3u playlists via the playlist button in latest commit. You can get the latest build from Actions tab to test it out.

**Comment by mtsbh** [comment]
Nice one! Thank you for the quick reaction and implementing this!
